src/NuGet.Clients/NuGet.Tools/GlobalSuppressions.cs

```diff
@@ -34,3 +34,4 @@
 [assembly: SuppressMessage("Build", "CA1031:Modify 'WriteUserOptions' to catch a more specific allowed exception type, or rethrow the exception.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetVSExtension.SolutionUserOptions.WriteUserOptions(Microsoft.VisualStudio.OLE.Interop.IStream,System.String)~System.Int32")]
 [assembly: SuppressMessage("Usage", "CA1816:Dispose methods should call SuppressFinalize", Justification = "IVsPackage.Close() method invokes overridden NuGetPackage.Dispose() method which internally invokes GC.SuppressFinalize() method.", Scope = "member", Target = "~M:NuGetVSExtension.NuGetPackage.Dispose(System.Boolean)")]
 [assembly: SuppressMessage("Design", "CA1001:Types that own disposable fields should be disposable", Justification = "IVsPackage.Close() method invokes overridden NuGetPackage.Dispose() method upon closing the IDE.", Scope = "type", Target = "~T:NuGetVSExtension.NuGetPackage")]
+[assembly: SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "<Pending>", Scope = "member", Target = "~M:NuGetVSExtension.NuGetPackage.CreateDocWindowForSolutionAsync(System.Boolean)~System.Threading.Tasks.Task{Microsoft.VisualStudio.Shell.Interop.IVsWindowFrame}")]
```

Comment:
This came up in the last review too. We should try to implement dispose instead of suppressing this.